tensorflow/core/ops/math_ops.cc

```diff
@@ -1380,10 +1380,23 @@ REGISTER_OP("HistogramFixedWidth")
     .Attr("T: {int32, int64, float32, float64}")
     .Attr("dtype: {int32, int64} = DT_INT32")
     .SetShapeFn([](InferenceContext* c) {
+      // value_range should be a vector.
```

Comment:
Can you add a unit test for the shape function here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/math_ops_test.cc